src/components/KeepReading/KeepReading.tsx

```diff
@@ -0,0 +1,67 @@
+'use client';
+import { StyledKeepReading } from './StyledKeepReading';
+import { Button } from '..';
+import { useNavigate } from 'react-router';
+
+// Navigation component for How To Play section. Creates four buttons, one for each How To Play section, except the current section. No Parameters. Uses URL to identify current section.
```

Comment:
Use the [JSDoc](https://jsdoc.app/about-getting-started) format for this. I believe there is a VS Code extension too that will autogenerate these for you. Put it on the line directly above the `KeepReading` function.

```suggestion
/**
 * Navigation component for How To Play section.
 * Creates four buttons, one for each How To Play section, except the current section. 
 * No Parameters. Uses URL to identify current section.
*/
```

Reply:
I don't love the idea of this component not having any props. It's great that the component can read the url, but we should do that kind of application logic in a "view" instead of a component. The view could read the url and alter the props that are passed down to `<KeepReading />`. The component should not have any understanding of its place in the site - all components should do is take in props and render them to the screen.

I know this probably seems tedious, but this makes the component harder to test.

Reply:
For example, you could create a Layout component for the how-to-play section that has this object:

```
    {
         Overview: '/how-to-play',
         Hosts: '/how-to-play/hosts',
         Contestants: '/how-to-play/contestants',
         Scoring: '/how-to-play/scoring',
         Lifelines: '/how-to-play/lifelines',
    }
```

This view component could read the current location using the `useNavigate` hook and filter out any of these keys that it doesn't need for the page (e.g., no "Overview" button on the Overview page). 

That kind of work can be done in a separate PR, but for now, let's have the `<KeepReading />` component accept some kind of prop object.